platform_test.go

```diff
@@ -55,11 +55,11 @@ func testPlatform(t *testing.T, context spec.G, it spec.S) {
 			bindings, err := libcnb.NewBindings("")
 			Expect(err).NotTo(HaveOccurred())
 
-			Expect(bindings).To(HaveLen(2))
-			types := []string{bindings[0].Type, bindings[1].Type}
-			Expect(types).To(ContainElements("elephantsql-type", "sendgrid-type"))
-			providers := []string{bindings[0].Provider, bindings[1].Provider}
-			Expect(providers).To(ContainElements("elephantsql-provider", "sendgrid-provider"))
+			Expect(bindings).To(HaveLen(3))
+			types := []string{bindings[0].Type, bindings[1].Type, bindings[2].Type}
+			Expect(types).To(ContainElements("elephantsql-type", "sendgrid-type", "postgres"))
+			providers := []string{bindings[0].Provider, bindings[1].Provider, bindings[2].Provider}
+			Expect(providers).To(ContainElements("elephantsql-provider", "sendgrid-provider", "postgres"))
```

Comment:
```suggestion
			Expect(providers).To(ContainElements("elephantsql-provider", "sendgrid-provider", "postgres"))
			
			// order of bindings can change, so sorting it for consistent test results
			slices.SortFunc(bindings, func(a, b libcnb.Binding) int {
				return cmp.Compare(a.Name, b.Name)
			})

			// verify regular values
			Expect(bindings[0].Secret).To(HaveKeyWithValue("bool", true))
			Expect(bindings[0].Secret).To(HaveKeyWithValue("int", 1))
			Expect(bindings[0].Secret).To(HaveKeyWithValue("uri", "postgres://exampleuser:examplepass@postgres.example.com:5432/exampleuser"))

			// verify with nested values
			Expect(bindings[2].Secret).To(HaveKeyWithValue("username", "foo"))
			Expect(bindings[2].Secret).To(HaveKeyWithValue("password", "bar"))
			Expect(bindings[2].Secret).To(HaveKeyWithValue("urls", `{"example":"http://example.com"}`))
```

I think things are working OK, but I'd like to see the values verified by reading the properties back and confirming the correct values/types. This will just help prevent potential regressions in the code.

Reply:
done